parser/src/main/scala/jawn/NullFacade.scala

```diff
@@ -25,7 +25,6 @@ object NullFacade extends Facade[Unit] {
   def jnull(): Unit = ()
   def jfalse(): Unit = ()
   def jtrue(): Unit = ()
-  def jnum(s: String): Unit = ()
-  def jint(s: String): Unit = ()
+  def jnum(s: String, decIndex: Int, expIndex: Int): Unit = ()
```

Comment:
Since the facade is a handler that will only ever be called by the parser, I'm not sure there's a need for the alternative single-parameter `jnum` or `jint`.

If we document the `s` arguments in the `Facade` API instead of here in `NullFacade` would that work?